### PR TITLE
Add Wiki Doc/Guidelines Links to Element Listings

### PR DESCRIPTION
**Pull request by talbs**
To better help folks find instructions and standards, this work adds some quick links to any elements in the UXPL that have a corresponding Github repo guidelines/styleguide wiki doc.

- - -

##### Reviewers

* [x] FED, Set up, and Overall Solution - @frrrances

### CODE REVIEW

File: _layouts/element.html [hunk 1]
```diff
@@ -16,7 +16,12 @@ <h1 class="pldoc-heading pldoc-element-title hd-4">{{ page.title }}</h1>
                 </li>
                 {% if page.example %}
                 <li class="meta meta-examplelink">
-                    <a class="pldoc-link" href="{{ site.exampleurl }}{{page.example}}"><span class="sr-only">View</span>Detailed Example</a>
+                    <a class="pldoc-link" href="{{ site.exampleurl }}{{ page.example }}"><span class="sr-only">View</span>Detailed Example</a>
```
Comment (frrrances) [inline _layouts/element.html]: Do you need a space in the span with "View" so it doesn't run together with "Detailed"?

File: _layouts/element.html [hunk 2]
```diff
@@ -16,7 +16,12 @@ <h1 class="pldoc-heading pldoc-element-title hd-4">{{ page.title }}</h1>
...
+                </li>
+                {% endif %}
+                {% if page.doc %}
+                <li class="meta meta-doclink">
+                    <a class="pldoc-link" href="{{ site.github.docurl }}{{ page.doc }}"><span class="sr-only">View</span>Documentation</a>
```
Comment (frrrances) [inline _layouts/element.html]: Do you need a space in the span with "View" so it doesn't run together with "Documenation"?

**Comment by frrrances** [comment]
A couple small nits but otherwise, w00t! :+1:

File: _layouts/element.html (hunk 2 above)
Comment (talbs) [inline _layouts/element.html]: Damn fine catch, I'm fixing this here and elsewhere (including the example link above this).

File: _layouts/element.html (hunk 1 above)
Comment (talbs) [inline _layouts/element.html]: Damn fine catch, I'm fixing this here and elsewhere (including the example link above this).

**Pull request merged**